contracts/ClaimSYNR.sol

```diff
@@ -14,22 +14,26 @@ contract ClaimSYNR is Ownable {
     ERC20 synr;
     bool public enabled;
     uint256 public award = 15000 * 10**18;
+    uint256 startBlock;
```

Comment:
Maybe it would be good to add an endBlock. At some moment we could decide to stop it. If we have that parameter we can use it.

Reply:
I will add it